lib/verifyLoggedInUser.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async (req, res, next) => {
```

Comment:
check login first time, I'm getting some weird not "quite" logged in behaviour???

Reply:
req.session.save() saves the session to the database for the next request... updated